### PR TITLE
change on_prem password

### PR DESCRIPTION
**Pull request by dorschw**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45768

## Description
Allow changing passwords for on-prem users.

## Does it break backward compatibility?
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphUser/ReleaseNotes/1_5_0.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Active Directory Users
+- Added the ***msgraph-user-change-password-on-premise*** command which changes the password of an on-premise user.
```
Comment (DeanArbel) [inline Packs/MicrosoftGraphUser/ReleaseNotes/1_5_0.md:4]:
```suggestion
- Added the ***msgraph-user-change-password-on-premise*** command. Change the password of an on-premise user.
```

**Review by DeanArbel — APPROVED** [review]
Nice

**Comment by tschanfeld** [comment]
reviewed only release note, as requested. no changes.

**Comment by dorschw** [comment]
TBC on https://github.com/demisto/content/pull/20581

**Pull request closed without merging**